### ISSUE TITLE
 Talkback - no labels on player buttons

### ISSUE THREAD

**Issue by melissafehr** [audioBoom/audioboo-android#3]
We've got a lot of blind users who rely on the talkback feature to navigate the app. The player buttons are accessible in 1.0.75, but are unlabelled and therefore really difficult to use.

Please add labels onto these.
